public/nextjs_migration/client/js/analytics.js

```diff
@@ -26,8 +26,16 @@ async function init({ debugMode, ga4MeasurementId }) {
     })
 
     // Instrument CTA clicks for analytics.
-    document.querySelectorAll('[data-cta-id]').forEach(cta =>
-      cta.addEventListener('click', () => window.gtag('event', 'clicked_cta', { cta_id: cta.dataset.ctaId })))
+    document
+      .querySelectorAll("[data-cta-id]")
```

Comment:
Q: why the switch from single quotes to double quotes? Or was this a Prettier thing (although if so, odd it didn't reformat the quotes on L8, 10, 21, 22, etc.

Reply:
Thanks - I just reset this branch and added a separate commit for the reformatting, so now it's consistent but the change should still be easy to see.